benchmarking/tpcds/__main__.py

```diff
@@ -0,0 +1,83 @@
+import argparse
+import logging
+from datetime import datetime
+from pathlib import Path
+
+import daft
+from daft.sql import SQLCatalog
+
+from . import datagen
+
+
+def generate_tpcds_catalog(tpcds_gen_folder: Path):
+    if not tpcds_gen_folder.exists():
+        raise RuntimeError
+    return SQLCatalog(
+        tables={
+            file.stem: daft.read_parquet(path=str(file))
+            for file in tpcds_gen_folder.iterdir()
+            if file.is_file() and file.suffix == ".parquet"
+        }
+    )
+
+
+def run_query(query_index: int, catalog: SQLCatalog) -> str:
+    sql_queries_path = Path("benchmarking") / "tpcds" / "queries"
+    query_file = sql_queries_path / f"{str(query_index).zfill(2)}.sql"
+    with open(query_file) as f:
+        query = f.read()
+
+    start = datetime.now()
+
+    try:
+        daft.sql(query, catalog=catalog).collect()
```

Comment:
Let's do a `.explain` before the `.collect()`? Also good to at least print the result after collecting.

Reply:
Updated.